app/models/intake_site_drop_off.rb

```diff
@@ -109,10 +109,16 @@ def phone_number=(value)
     end
   end
 
+  # Returns the phone number formatted for user display, e.g.: "(510) 555-1234"
   def formatted_phone_number
     Phonelib.parse(phone_number).local_number
   end
 
+  # Returns the phone number in the E164 standardized format, e.g.: "+15105551234"
+  def standardized_phone_number
+    Phonelib.parse(phone_number).e164
+  end
```

Comment:
If `phone_number` is `nil`, this would just return `nil`, correct?

Reply:
Yep, though I realized I didn't explicitly add a test for that. I'll push one up in this branch just to be sure.